winui/Rating/Customization.md

```diff
@@ -1,12 +1,254 @@
 ---
 layout: post
-title: Customization with WinUI Rating control | Syncfusion
-description: Learn here all about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more.
+title: Customization in WinUI Rating control | Syncfusion
+description: Learn here all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control.
```

Comment:
description: Learn all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control here.

Reply:
Changed the content as mentioned above.